Snowflake.Data.Tests/UnitTests/SFSessionPropertyTest.cs

```diff
@@ -192,7 +192,41 @@ public static IEnumerable<TestCase> ConnectionStringTestCases()
                     { SFSessionProperty.CHANGEDSESSION, DefaultValue(SFSessionProperty.CHANGEDSESSION) },
                     { SFSessionProperty.WAITINGFORIDLESESSIONTIMEOUT, DefaultValue(SFSessionProperty.WAITINGFORIDLESESSIONTIMEOUT) },
                     { SFSessionProperty.EXPIRATIONTIMEOUT, DefaultValue(SFSessionProperty.EXPIRATIONTIMEOUT) },
-                    { SFSessionProperty.POOLINGENABLED, DefaultValue(SFSessionProperty.POOLINGENABLED) }
+                    { SFSessionProperty.POOLINGENABLED, "false" } // by default pooling is disabled for externalbrowser authentication
+                }
+            };
+            var testCaseWithExternalBrowserAndPoolingEnabled = new TestCase()
```

Comment:
it works, and we have some tests written in this manner, but I would prefer the way of writing test like TestValidateCorrectAccountNames() - so a dedicated test verifying only the properties which are important from given test point of view.